fintorch/layers/explainable/relevancepropagation.py

```
@@ -0,0 +1,38 @@
+    The result of a / b, with 0 where b was 0.
+  """
+  # Create a tensor of zeros with the same shape and type as 'a'
+  zeros = torch.zeros_like(a)
+
+  # Where b is not zero, compute a/b. Otherwise, use the 'zeros' tensor.
+  return torch.where(b != 0, a / b, zeros)
+
+
+class LinearRelevancePropagation(nn.Module):
+    def __init__(self):
+        super(LinearRelevancePropagation, self).__init__()
```

Pylint (3.3.7)

[refactor] Consider using Python 3 style super() without arguments

(R1725)